src/extension.ts

```diff
@@ -507,9 +507,7 @@ function launchMetals(
     )
   );
 
-  context.subscriptions.push(client.start());
-
-  return client.onReady().then(
```

Comment:
https://github.com/microsoft/vscode-languageserver-node/blob/95e411de3d10c0fa1e0371b84e88f482682a9170/README.md#3170-protocol-800-json-rpc-800-client-and-800-server